src/python/bindings/bind.cpp

```diff
@@ -54,6 +52,7 @@ PYBIND11_MODULE(smartredisPy, m) {
         .def(py::init<std::string&>());
 
     // Python client class
+    #define CLIENT_METHOD(name)  def(#name, &PyClient::name)
```

Comment:
Is this macro used? (Like the idea of using it)

Reply:
Not yet -- it will be a separate PR to switch over to it